Sources/WebSocket/WebSocket+Client.swift

```diff
@@ -31,9 +31,10 @@ extension HTTPClient {
         hostname: String,
         port: Int? = nil,
         path: String = "/",
+        headers: HTTPHeaders = .init(),
```

Comment:
docblock needs to be adjusted for this new param

Reply:
done 🙂